integration-tests/capacity/transactions.test.ts

```diff
@@ -127,73 +181,49 @@ describe("Capacity Transaction Tests", function () {
             // Confirm that the staked capacity was removed from the stakeProviderId account using the query API
             const capacityStaked = (await firstValueFrom(ExtrinsicHelper.api.query.capacity.capacityLedger(stakeProviderId))).unwrap();
             assert.equal(capacityStaked.remainingCapacity,   0, "should return a capacityLedger with 0 remainingCapacity");
-            assert.equal(capacityStaked.totalTokensStaked,   0,       "should return a capacityLedger with 0 total tokens staked");
-            assert.equal(capacityStaked.totalCapacityIssued, 0,       "should return a capacityLedger with 0 capacity issued");
+            assert.equal(capacityStaked.totalTokensStaked,   0, "should return a capacityLedger with 0 total tokens staked");
+            assert.equal(capacityStaked.totalCapacityIssued, 0, "should return a capacityLedger with 0 capacity issued");
         });
     });
 
     describe("increase stake testing", function () {
         it("should successfully increase stake for stakeProviderId", async function () {
             // Now starting from zero again with stakeProviderId, Stake 1M capacity
-            const stakeObj = ExtrinsicHelper.stake(stakeKeys, stakeProviderId, 1000000);
-            const [stakeEvent] = await stakeObj.fundAndSend();
-            assert.notEqual(stakeEvent, undefined, "should return a Staked event");
-
-            if (stakeEvent && ExtrinsicHelper.api.events.capacity.Staked.is(stakeEvent)) {
-                let stakedCapacity = stakeEvent.data.capacity;
-                assert.equal(stakedCapacity, 1000000, "should return a Staked event with 1M capacity");
-            }
+            await stakeToProvider(stakeKeys, stakeProviderId, 1000000n);
```

Comment:
This test is kind of weird to me, and the abstraction to stakeToProvider shows it in a glaring way. If we're simply testing a staking increase, it doesn't really matter whether we start from capacity=0.  We only need to call stakeToProvider once.

Maybe it's worth a test staking to the same provider from two different MSAs?